src/components/Modal/Modal.js

```diff
@@ -50,7 +50,8 @@ class Modal extends Component {
       className,
       sparkComponentClassName('Modal'),
       sparkClassName('utility', 'Display', null, 'none'),
-      sparkClassName('component', 'Modal', null, type)
+      sparkClassName('component', 'Modal', null, type),
```

Comment:
`sparkComponentClassName` is an easier function to use for this.

```suggestion
      sparkComponentClassName('Modal', null, type),
```

Reply:
yep!